src/zope/app/apidoc/interface.rst

```diff
@@ -227,7 +224,7 @@ This function returns a page-template-friendly dictionary for a method:
 
   >>> pprint(interface.getMethodInfoDictionary(IFoo['blah'])) #doc
   {'doc':
-     u'<p>This is the <cite>blah</cite> method.</p>\n',
+     u'<p>This is the `blah` method.</p>\n',
```

Comment:
Ah, so it comes down to a difference in utility registrations.

The default document format is actually structuredtext (STX) not *re*structuredtext, so that's what's rendering this...at least in theory. But the old test setup code explicitly overrode that:
```python
    ztapi.provideUtility(IFactory, ReStructuredTextSourceFactory,
                         'zope.source.stx')
```
So it was being rendered as rst.

The new setup actually configures `zope.app.renderer`, so it's now being rendered as stx. That's easily fixed, though.